types/postcss-env-function/index.d.mts

```diff
@@ -0,0 +1,2 @@
+import postcssEnvFunction = require("./index");
```

Comment:
I am sorta shocked that this works. Isn't this an mts file? Shouldn't this be:

```ts
import postcssEnvFunction from "./index.js";
export default postcssEnvFunction
```

To trigger CJS resolution from MJS?

Reply:
Wow, still passes in node16. Confusing.

Reply:
Still passes, wat

Reply:
Not really sure what’s happening either, but I can still replace the contents with the snippet you suggested